include/proxy-wasm/context.h

```diff
@@ -245,6 +245,19 @@ class ContextBase : public RootInterface,
 #else
     unimplemented();
     return 0;
+#endif
+  }
+  uint64_t getMonotonicTimeNanoseconds() override {
+#if !defined(_MSC_VER)
+    struct timespec tpe;
+    clock_gettime(CLOCK_MONOTONIC, &tpe);
+    uint64_t t = tpe.tv_sec;
+    t *= 1000000000;
+    t += tpe.tv_nsec;
+    return t;
+#else
+    unimplemented();
+    return 0;
```

Comment:
I guess this is fine for now, since it follows `getCurrentTimeNanoseconds()`, but we should remove those default implementations from headers, to avoid accidentally leaking them into production code... and then we can have a dummy `Context` implementation used for tests in this repo.

Reply:
OK, will do the cleanup in https://github.com/proxy-wasm/proxy-wasm-cpp-host/issues/158 later, so I'm leaving this as-is for now.